packages/toast/src/toast-pending/ToastPending.tsx

```diff
@@ -0,0 +1,20 @@
+import { toast, ToastContent, ToastOptions } from 'react-toastify'
+import {
+  ToastPendingLoaderStyle,
+  ToastPendingStyle,
+  ToastPendingTextStyle,
+} from './ToastPendingStyles'
+import { TOASTS_PENDING_OPTIONS } from '../toastsDefaultOptions'
+
+export function ToastPending(
```

Comment:
because:
1) he is different
2) i use component an approach (ToastPending use styled-components and @lidofinance/loaders)
3) many small files are good, one large is bad

Reply:
Yes, I completely agree that it is better to split components into separate files. I would also do this with the rest of the components in the `Toast.tsx`. So, it's not really important, let's leave it as it is. I resolved the conversation.